src/api/public/apidocs-new/paths/source_project_name_config.yaml

```diff
@@ -0,0 +1,55 @@
+get:
+  summary: Get project's configurations
+  description: |
+    Read configurations for the specified project
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+    - name: rev
+      in: query
+      schema:
+        type: string
+  responses:
+    '200':
```

Comment:
If I force to raise any of the exceptions in [this line](https://github.com/openSUSE/open-build-service/blob/e05b4437d9c73f7ddba828bcf8eec2b999b41a95/src/api/app/controllers/source_project_config_controller.rb#L62), I get a 404 response. I think it should be documented.

Reply:
I'm not able to reproduce 404 even for an unknown project. Because I receive 406 `Not Acceptable`. This is the same behavior with almost every other case except 200. But looking at the code it should produce 404. I've added 404 in expected responses

Reply:
When the project is remote it should raise the 404 one.